AUTHORS

```diff
@@ -120,6 +120,7 @@ answer newbie questions, and generally made Django that much better:
     Brandon Chinn <http://brandonchinn178.github.io>
     Brant Harris
     Brendan Hayward <brendanhayward85@gmail.com>
+    Brendan Quinn <brendan@cluefulmedia.com>
```

Comment:
@bquinn You [submitted the CLA](https://www.djangoproject.com/foundation/cla/) too yes?

Reply:
Yes and @freakboy3742 confirmed receipt of the signed CLA by email on 10 Jan.